### ISSUE TITLE
[Bug] Mvit_small from mmaction2 Inference speed onnx vs torch

### ISSUE THREAD

**Issue by nagadit** [open-mmlab/mmdeploy#1875]
### Checklist

- [X] I have searched related issues but cannot get the expected help.
- [X] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/faq.md) but cannot get the expected help.
- [ ] 3. The bug has not been fixed in the latest version.

### Describe the bug

![image](https://user-images.githubusercontent.com/24617421/224991796-92c5363c-ee95-47ff-a440-f0892d06ce7d.png)

### Reproduction

model converted using torch2onnx script

### Environment

```Shell
TorchVision: 0.12.0
OpenCV: 4.5.4
MMEngine: 0.6.0
MMCV: 2.0.0rc4
MMCV Compiler: GCC 9.3
MMCV CUDA Compiler: 11.3
MMDeploy: 1.0.0rc3+ec4abe5


**********Backend information**********
tensorrt:    None
ONNXRuntime: 1.14.1
ONNXRuntime-gpu:     None
ONNXRuntime custom ops:      Available
pplnn:       None
ncnn:        None
snpe:        None
openvino:    2022.3.0
torchscript: 1.11.0
torchscript custom ops:      Available
rknn-toolkit:        None
rknn-toolkit2:       None
ascend:      None
coreml:      None
tvm: None
vacc:        None


**********Codebase information**********
mmdet:       None
mmseg:       None
mmcls:       None
mmocr:       None
mmedit:      None
mmdet3d:     None
mmpose:      None
mmrotate:    None
mmaction:    1.0.0rc3
```

**Comment by irexyc** [comment]
Hi, @nagadit 
Sorry for late reply, I can't reproduce your result.

I connvert the this model to onnx, modified some parameters to set the input with `1, 3, 16, 224, 224` dim, and tested with onnx and pytorch.

Below is my test script.
```
import onnxruntime as ort
import torch
import time

from mmengine.dataset import Compose, pseudo_collate
from mmaction.apis import inference_recognizer, init_recognizer
from mmengine import Config



# Choose to use a config and initialize the recognizer
config = '/root/ws/deploee/mmaction2/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py'
config = Config.fromfile(config)
# Setup a checkpoint file to load
checkpoint = '/root/ws/deploee/mmdeploy_checkpoints/mmaction/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb_20220901-701b0f6f.pth'
# Initialize the recognizer
model = init_recognizer(config, checkpoint, device='cpu').eval()

test_pipeline_cfg = config.test_pipeline
test_pipeline = Compose(test_pipeline_cfg)



sess = ort.InferenceSession('opt.onnx')
input = torch.randn(1,1,3,16,224,224)
input_feed = {
    'input': input.cpu().data.numpy()
}

data = dict(filename='/root/ws/deploee/mmaction2/demo/demo.mp4', label=-1, start_index=0, modality='RGB')
data = test_pipeline(data)
data = pseudo_collate([data])


def pytorch_run():
    t1 = time.perf_counter()
    with torch.no_grad():
        for _ in range(50):
            out = model.test_step(data)
    t2 = time.perf_counter()
    print(t2 - t1)
    


def onnx_run():
    t1 = time.perf_counter()
    for _ in range(50):
        out = sess.run(['output'], input_feed=input_feed)
    t2 = time.perf_counter()
    print(t2 - t1)

```

And the result on my machine is 

```python
>>> pytorch_run()
7.970148514024913
>>> onnx_run()
5.931804160121828
>>> pytorch_run()
7.131746815983206
>>> onnx_run()
5.9229006241075695
>>> pytorch_run()
7.57818131102249
>>> onnx_run()
5.8941806689836085
```

**Comment by nagadit** [comment]
Hi @irexyc ! Thanks for your answer! Can you show how you converted this model? There may be a problem in the conversion pipeline.

**Comment by irexyc** [comment]
I modify the config to 
```diff
(torch1.10.0) root@0c52991efc18:~/ws/deploee/mmaction2# git diff configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py
diff --git a/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py b/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics40
0-rgb.py
index 7c3c0a66..48fbcedb 100644
--- a/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py
+++ b/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py
@@ -39,13 +39,13 @@ test_pipeline = [
     dict(type='DecordInit', **file_client_args),
     dict(
         type='SampleFrames',
-        clip_len=32,
+        clip_len=16,
         frame_interval=2,
-        num_clips=10,
+        num_clips=1,
         test_mode=True),
     dict(type='DecordDecode'),
     dict(type='Resize', scale=(-1, 256)),
-    dict(type='ThreeCrop', crop_size=256),
+    dict(type='CenterCrop', crop_size=224),
     dict(type='FormatShape', input_format='NCTHW'),
     dict(type='PackActionInputs')
 ]
```

and use this command
``` bash
python tools/deploy.py configs/mmaction/video-recognition/video-recognition_onnxruntime_static.py \ 
../mmaction2/configs/recognition/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb.py \ 
../mmdeploy_checkpoints/mmaction/slowfast/slowfast_r50_8xb8-4x16x1-256e_kinetics400-rgb_20220901-701b0f6f.pth \ 
../mmaction2/demo/demo.mp4 \ 
--work-dir onnx \ 
--dump-info
```

**Comment by nagadit** [comment]
Fixed, my bad) Thanks!
